chore: remove `__SVELTEKIT_DEV__` global (#14308)
sveltejs/kit: .changeset/true-moles-tap.md

---
'@sveltejs/kit': patch
---

> chore: consolidate dev checks to `esm-env` and remove the `__SVELTEKIT_DEV__` global

Author: ```suggestion
chore: consolidate dev checks to use `esm-env` instead of a `__SVELTEKIT_DEV__` global
```
